src/System.Collections/tests/Generic/HashSet/HashSet.Generic.Tests.cs

```diff
@@ -344,5 +344,19 @@ public void CanBeCastedToISet()
             ISet<T> iset = (set as ISet<T>);
             Assert.NotNull(iset);
         }
+
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
+        public void EnsureCapacity_AfterRemovingOne(int setLength)
+        {
+            if (setLength == 0)
+            {
+                return;
+            }
+
+            HashSet<T> set = (HashSet<T>)GenericISetFactory(setLength);
+            set.Remove(set.ElementAt(0));
+            set.EnsureCapacity(set.Count * 3); // * 3 to ensure we hit the next prime so that a resize happens.
```

Comment:
Is this a known property of primes or just tested happenstance?

Reply:
Eg. 
prime := 2 => 2 * 3 = 6, which is not a prime
prime := 3 => 3 * 3 = 9, which is not a prime
...
prime := 4001 => 4001 * 3 = 

Bertrand's postulate states something different.